schema.prisma

```diff
@@ -486,6 +486,30 @@ model BankAccount {
   @@map("bank_accounts")
 }
 
+// Stores all movements for Podkrepi.bg IBAN 
+model BankTransaction {
+  id                 String              @id @unique @db.VarChar(250)
+  /// IBAN number of the account tracked
+  ibanNumber         String              @db.VarChar(34)
+  /// Bank name
+  bankName           String              @map("bank_name") @db.VarChar(50)
+  /// Bank Identification Code, BIC/SWIFT code
+  bankIdCode         String              @map("bank_id_code") @db.VarChar(50)
+  transactionDate    DateTime            @map("transaction_date") @db.Timestamptz(6)
+  senderName         String?             @map("sender_name") @db.VarChar(100)
+  recipientName      String?             @map("recipient_name") @db.VarChar(100)
+  senderIban         String?             @db.VarChar(34)
+  recipientIban      String?             @db.VarChar(34)
```

Comment:
We might add a `@map` for these two files as well so they are presented as snake case in the db schema.

```suggestion
  senderIban         String?             @map("sender_iban") @db.VarChar(34)
  recipientIban      String?             @map("recipient_iban") @db.VarChar(34)
```

Reply:
Same for `bankDonationStatus`

Reply:
Agreed! I've refactored it to snake_case